build.sh

```diff
@@ -49,7 +88,7 @@ esac
 FILENAME="$TARGET-$VERSION"
 
 echo ""
-echo "$(tput setaf 2)Bootstrapping dependencies using carthage$(tput sgr 0)"
```

Comment:
Please put back color

Reply:
That is I want to leave to the designer. I think you want white. But me and you are just coder, so designer can better decide on this

Reply:
I have put back white color